src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
```

Comment:
This class is generally well written. The methods dont really need a javaDocs since they are easy to understand but it will be better to add a short one:)